### PR TITLE
CART-958 CartTest: Remove memory leak from cart rpc one/two node test.

### PR DESCRIPTION
**Pull request by SABollinger**
Signed-off-by: Steven Bollinger <stevenx.bollinger@intel.com>

### CODE REVIEW

**Comment by SABollinger** [comment]
Looking for code review from @frostedcmos  @jgmoore-or @sylviachanoiyee @mallove79

**Review by frostedcmos — APPROVED** [review]

**Comment by SABollinger** [comment]
Still waiting for a code review from @jgmoore-or , @mallove79, @sylviachanoiyee

**Review by jgmoore-or — APPROVED** [review]

**Pull request merged**
